ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -316,7 +316,7 @@ class ErrorFactories private (errorCodesVersionSwitcher: ErrorCodesVersionSwitch
         .asGrpcError,
     )
 
-  def invalidDeduplicationDuration(
+  def invalidDeduplicationPeriod(
```

Comment:
Is this because you're going to use it for other types of deduplication periods? Could you record the rationale in the PR
s description, so that it's clearer for posterity (likely ourselves)?

Reply:
The rationale is mostly to have it in sync with the error code used. 
Will change the description